index.html

```diff
@@ -5,20 +5,21 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="./font/stylesheet.css">
     <title>Portfolio</title>
 </head>
 
-<body>
+<body> 
     <div class="container">
         <div class="elements">
             <p class="item logo">My logo</p>
-            <div class="item harmburger"><img src="/Graphics/" alt=""></div>
+            <div class="item harmburger"><img src="./Graphics/Normal Button/Tertiary/Icons/Union.svg" alt=""></div>
 
         </div>
 
         <h1 class="greeting">
-            I’m Caleb
+            I’m Caleb<br>
             Glad to see you!
         </h1>
 
```

Comment:
- [ ] Kindly wrap your `greeting`, `connect`, and `social` elements with a `section` tag.
  - As, it represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
  - Here is an example below that you can consider.
```
<section>
        <h1 class="greeting">
            I’m Caleb<br>
            Glad to see you!
        </h1>
        <h3 class="connect">
            LET’S CONNECT
        </h3>
        <ul class="social">
           <li class="icon1">
                 <a href="https://twitter.com"><img src="twitter.jpg" alt=""/></a>
            </li>
            <li class="icon2">
                 <a href="https://github.com"><img src="github.jpg" alt=""/></a>
            </li>
         </ul>
</section>
```